TasksFromYurijL/src/SimpleTasks/NumberCheck.java

```diff
@@ -0,0 +1,18 @@
+package SimpleTasks;
+
+public class NumberCheck {
+	/**
+	 *  @return Check if the number is odd
+	*/
+	public boolean isOddNumber(int number) {
+		if(0!=number%2) return true;

```

Comment:
you do not have to use if operator to assigning boolean expression, please try to make this in one line, the same for even

Reply:
also you can try to use modulus operators, which is fastest than dividing